presto-sqlserver/src/main/java/com/facebook/presto/plugin/sqlserver/SqlServerClient.java

```diff
@@ -35,16 +35,19 @@ public SqlServerClient(JdbcConnectorId connectorId, BaseJdbcConfig config)
             throws SQLException
     {
         super(connectorId, config, "\"", new SQLServerDriver());
+        connectionProperties.setProperty("initializationString", "SET QUOTED_IDENTIFIER, ANSI_NULLS, CONCAT_NULL_YIELDS_NULL, ARITHABORT ON;");
+        connectionProperties.setProperty("sendStringParametersAsUnicode", "false");
+        connectionProperties.setProperty("applicationName", "Presto");
     }
 
     @Override
     public void commitCreateTable(JdbcOutputTableHandle handle)
     {
         StringBuilder sql = new StringBuilder()
                 .append("sp_rename ")
-                .append(singleQuote(handle.getCatalogName(), handle.getSchemaName(), handle.getTemporaryTableName()))
+                .append("'" + quoted(handle.getCatalogName(), handle.getSchemaName(), handle.getTemporaryTableName()) + "'")
```

Comment:
Yeah well that's a weird one with SQL Server, then end result has to look something like this:
```
EXEC sp_rename '[northwind].[dbo].[some [weird]] table / name]', 'new_name'
```
You have to quote the fully qualified table name and then pass that as a string literal. It also appears to work with `"`-quoting, but I haven't done extensive testing with it.